test/stdlib/StringFlatMap.swift

```diff
@@ -1,23 +1,14 @@
 // RUN: %empty-directory(%t)
-// RUN: %target-build-swift %s -o %t/a.out3 -swift-version 3 && %target-run %t/a.out3
 // RUN: %target-build-swift %s -o %t/a.out4 -swift-version 4 && %target-run %t/a.out4
 
 // REQUIRES: executable_test
 
 import StdlibUnittest
 
-#if swift(>=4)
 
 public typealias ExpectedResultType = [Character]
 let swiftVersion = "4"
```

Comment:
Nit: I think we should remove these two variables now; they make it slightly harder to understand the tests.